Dockerfile

```diff
@@ -69,6 +69,8 @@ FROM base AS runtime
 
 USER forseti
 
+ENV PORT 50051
```

Comment:
Does this mean that PORT variable is 50051? Or that default is 50051?

Reply:
What this does is expose the environment variable to be set by the container run time.  If it is not set there (e.g. not set via Kubernetes or Cloud Run), then it defaults to 50051.

I set a default in two places because I am being overly cautious.

Reply:
Thanks for the explanation!